docker/README

```diff
@@ -0,0 +1,37 @@
+About:
+
+AndBible's build system requires a newer version of nodejs than is packaged
+for Ubuntu 20.04.  As a result I can no longer build AndBible.  I created
+this Docker image based on Ubuntu 22.04 so that I can use it to build
+AndBible.
```

Comment:
Just a tip: you can install any node versions on user level with nvm https://github.com/nvm-sh/nvm